### PR TITLE
Enable configuration of AV1 screen content coding tools

### PR DESCRIPTION
**Pull request by aboba**
Fixes https://github.com/w3c/webcodecs/issues/646

### CODE REVIEW

File: av1_codec_registration.src.html [hunk 1]
```diff
@@ -73,6 +73,45 @@
...
+    Contains codec specific configuration options for the AV1 codec.
+  </dd>
+</dl>
+
+AV1EncoderConfig {#av1-encoder-config}
+--------------------------------------
+<pre class='idl'>
+<xmp>
+dictionary AV1EncoderConfig {
+ boolean seqForceScreenContentTools = false
```
Comment (chrisn) [inline av1_codec_registration.src.html:99]: Why the `seq` prefix - could this be `forceScreenContentTools`?

Comment (aboba) [reply av1_codec_registration.src.html]: Fixed.

File: av1_codec_registration.src.html [hunk 2]
```diff
@@ -73,6 +73,46 @@
...
+    Contains codec specific configuration options for the AV1 codec.
+  </dd>
+</dl>
+
+AV1EncoderConfig {#av1-encoder-config}
+--------------------------------------
+<pre class='idl'>
+<xmp>
+dictionary AV1EncoderConfig {
+ boolean forceScreenContentTools = false
```
Comment (dalecurtis) [inline av1_codec_registration.src.html:99]: space.

**Review by dalecurtis — APPROVED** [review]

File: av1_codec_registration.src.html (hunk 2 above)
Comment (aboba) [reply av1_codec_registration.src.html]: Fixed.

**Comment by aboba** [comment]
Rebased in PR https://github.com/w3c/webcodecs/pull/662

**Pull request closed without merging**